An error occurred while executing the following cell:
------------------
# Least confidence uncertanity
uncertainty_lc = 1 - np.maximum.reduce((prob_iris_setosa, prob_iris_versicolor, prob_iris_virginica))

plot_data(
    prob_iris_setosa,
    prob_iris_versicolor,
    prob_iris_virginica,
    uncertainty_lc,
    "Least Confidence Sampling",
    save_name="Least Confidence Sampling Ternary Countor",
)
------------------

---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
/tmp/ipykernel_4502/713680158.py in <module>
      8     uncertainty_lc,
      9     "Least Confidence Sampling",
---> 10     save_name="Least Confidence Sampling Ternary Countor",
     11 )

/tmp/ipykernel_4502/3512797537.py in plot_data(prob0, prob1, prob2, uncertanity, title, save_name)
      9         colorscale="Jet",
     10         showscale=True,
---> 11         title=title,
     12     )
     13 

~/miniconda3/envs/py37/lib/python3.7/site-packages/plotly/figure_factory/__init__.py in create_ternary_contour(*args, **kwargs)
     45 
     46     def create_ternary_contour(*args, **kwargs):
---> 47         raise ImportError("Please install scikit-image to use `create_ternary_contour`")
     48 
     49 

ImportError: Please install scikit-image to use `create_ternary_contour`
ImportError: Please install scikit-image to use `create_ternary_contour`
